java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -92,10 +95,24 @@ public static BaseTransaction<?> transactionTest(BaseTransaction<?> baseTransact
 
         baseTransaction.configResetStat();
 
-        // TODO
-        // baseTransaction.pfadd(hllKey1, new String[0]);
-
-        // TODO pfcount
+        baseTransaction
+                .pfadd(hllKey1, new String[0])
```

Comment:
We should have just one test for transactions we can test functionality in shared commands. (This is consistent with Py and Node and Java)